### ISSUE TITLE
broken phpdoc for closures

### ISSUE THREAD

**Issue by 0x450x6c** [prettier/plugin-php#1425]
**@prettier/plugin-php v0.14.0**
[Playground link](https://loilo.github.io/prettier-php-playground/#N4IgDgTgpgLjCWUIHkwIPYDsDOIBcok8mMA6vACYwAW+AHAAwA0IMAhgEblW14AsLAK7YoAFU648AMzYAbES2zEA5rKgBFQehhR8M+VBZhqYAGpIlWfCADsAOj4gWMCG3iyVAYXQBbH2wAFAAkA-BdBQxAOVwBjKABlGABPNWswbAgAWgAmJxBoAEdBeGgA12V-PTkFEGIRCBgytgq2KoMAXxYoCngYdBQ0eCxJUDZsGDCICM7azDBBCbwQAB4AfmMwAB1MbYASKXR0AAIAXiOAegAqS+3Lo4ABMDZXHyPn1yTgNjwj4hgmI4cH7jCAqdpHXYcZ63c7bcZsBAxI5STBHAAU7zYSQhUIgAEpTgA+HHPADcIHaQA)

**Input:**
```php
<?php

$foo = /**
* @param array{a: int, b: string} $bar
*/
static fn (array $bar) => $bar;
```

**Output:**
```php
<?php

$foo /**
 * @param array{a: int, b: string} $bar
 */ = static fn(array $bar) => $bar;

```

**Expected:**
```php
<?php

$foo = /**
* @param array{a: int, b: string} $bar
*/
static fn (array $bar) => $bar;
```

Hi, I am using psalm, and currently it is not possible to use psalm annotations for closures due to this bug.

**Comment by czosel** [comment]
I see, thanks for reporting this!

**Comment by czosel** [comment]
We just released `v0.14.2` which fixes this :tada:
